dask/array/tests/test_fft.py

```diff
@@ -73,6 +123,20 @@ def test_irfft():
     assert_eq(irfft(darr2, axis=0), np.fft.irfft(nparr, axis=0))
 
 
+def test_irfft2():
+    assert_eq(irfft2(darr3), np.fft.irfft2(nparr))
+    assert_eq(irfft2(darr3, axes=(1, 0)), np.fft.irfft2(nparr, axes=(1, 0)))
+    assert_eq(irfft2(darr3, (8, 9), axes=(1, 0)),
+              np.fft.irfft2(nparr, (8, 9), axes=(1, 0)))
+
+
+def test_irfftn():
+    assert_eq(irfftn(darr3), np.fft.irfftn(nparr))
+    assert_eq(irfftn(darr3, axes=(1, 0)), np.fft.irfftn(nparr, axes=(1, 0)))
+    assert_eq(irfftn(darr3, (8, 9), axes=(1, 0)),
+              np.fft.irfftn(nparr, (8, 9), axes=(1, 0)))
+
```

Comment:
Any thoughts on pulling in all of these tests to a large parametrized test that was more comprehensive?